pkg/broker/catalog.go

```diff
@@ -0,0 +1,46 @@
+package broker
+
+import (
+	"context"
+
+	"github.com/pivotal-cf/brokerapi"
+
+	"github.com/presslabs/gitea-service-broker/pkg/cmd/options"
+)
+
+var sharedPlan = brokerapi.ServicePlan{
+	ID:          options.SharedPlanID,
+	Name:        options.SharedPlanName,
+	Description: "This plan provides a Gitea repository on a shared VM configured for data persistence.",
+	Metadata: &brokerapi.ServicePlanMetadata{
+		Bullets: []string{
+			"Each instance shares the same VM",
+			"Single dedicated Gitea process",
+			"Suitable for development & testing workloads",
+		},
+		DisplayName: "Shared-VM",
```

Comment:
See the comment regarding `shared` naming. It should also state something like `Repository` rather than `Shared-...`.